src/components/Main/Main.html

```diff
@@ -4,31 +4,39 @@
       <div class="mdc-list">
         <a class="mdc-list-item mdc-list-item--activated" href="#" aria-selected="true">
           <i class="material-icons mdc-list-item__graphic" aria-hidden="true">library_music</i>
-          <span class="mdc-list-item__text mdc-typography--headline4">My Songs</span>
+          <span class="mdc-list-item__text mdc-typography--headline5">My Songs</span>
         </a>
         <a class="mdc-list-item" href="#">
           <i class="material-icons mdc-list-item__graphic" aria-hidden="true">album</i>
-          <span class="mdc-list-item__text mdc-typography--headline4">Albums</span>
+          <span class="mdc-list-item__text mdc-typography--headline5">Albums</span>
         </a>
         <a class="mdc-list-item" href="#">
           <i class="material-icons mdc-list-item__graphic" aria-hidden="true">person_outline</i>
-          <span class="mdc-list-item__text mdc-typography--headline4">Artists</span>
+          <span class="mdc-list-item__text mdc-typography--headline5">Artists</span>
         </a>
         <a class="mdc-list-item" href="#">
           <i class="material-icons mdc-list-item__graphic" aria-hidden="true">info</i>
-          <span class="mdc-list-item__text mdc-typography--headline4">About</span>
+          <span class="mdc-list-item__text mdc-typography--headline5">About</span>
         </a>
       </div>
     </div>
   </aside>
   <div class="main__wrapper mdc-drawer-app-content">
-    <header class="main__header mdc-top-app-bar app-bar"></header>
+    <header class="mdc-top-app-bar app-bar">
+      <div class="main__header"></div>
+      <div class="main__search"></div>
+    </header>
     <main class="main__section main-content">
       <div class="main__content-mount mdc-top-app-bar--fixed-adjust">
-        <h5 class="mdc-typography--headline3">Albums</h5>
+        <h3 class="mdc-typography--headline4">Albums
+        </h3>
+        Albums <br>Albums <br>Albums <br>Albums <br>Albums <br>Albums <br>Albums <br>Albums <br>Albums <br>Albums <br>Albums
```

Comment:
???

Reply:
Just a sample content :-) I will remove it